examples/plugins/src/knip/README.md

```diff
@@ -0,0 +1,89 @@
+# knip
+
+🕵️ **Code PushUp plugin for detecting unused code** 📋
+
+---
+
+## Getting started
+
+1. If you haven't already, install [@code-pushup/cli](../cli/README.md) and create a configuration file.
+
+2. Copy the [plugin source](../file-size) as is into your project
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** Wrong name here‏